gravitee-apim-console-webui/src/management/api/runtime-logs-v4/api-runtime-logs.component.html

```diff
@@ -0,0 +1,41 @@
+<!--
+
+    Copyright (C) 2015 The Gravitee team (http://gravitee.io)
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+            http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+<ng-container>
+  <div class="runtime-logs__header">
+    <div>
+      <span class="mat-h2">Runtime Logs</span>
+    </div>
+    <div>
+      <span class="mat-h5">Debug and Optimize your API by displaying logs from your API runtime activities</span>
```

Comment:
The colors and fonts are not the good one I think. I would create a class like this 

```scss

.runtime-logs__header {
  margin-bottom: 24px;

  &__subtitle {
    color: mat.get-color-from-palette(gio.$mat-space-palette, 'lighter40');
    @include mat.typography-level($typography, 'mat-body');
  }
}

```